account_invoice_import_invoice2data/wizard/account_invoice_import.py

```diff
@@ -76,7 +77,16 @@ def invoice2data_to_parsed_inv(self, invoice2data_res):
 
         for line in lines:
             # Manipulate line data to match with account_invoice_import
-            line['price_unit'] = float(line.get('price_unit', 0))
+            price_unit = line.get('price_unit', 0)
+
+            # remove thousand seperators (, OR .)
+            price_unit = re.sub(r'\.(?=[\d,]*\,\d{2}\b)', '', price_unit)
+            price_unit = re.sub(r',(?=[\d,]*\.\d{2}\b)', '', price_unit)
```

Comment:
Assuming two decimals is not generic. Have you got the currency available here? Then you can make a more informed choice about the number of decimals.

Reply:
Changed